### PR TITLE
Update Workflow

### PR DESCRIPTION
**Pull request by donavanbecker**
# Name of the PR

## :recycle: Current situation
Currently using stale App

## :bulb: Proposed solution
Switch to stale GitHub Actions

### Problem that is solved
More reliable then App

### Implications
Remove Apps

## :heavy_plus_sign: Additional Information
We would just need to Remove Apps Afterwards

### Testing
Currently have this on my plugins.

### Reviewer Nudging
@Supereg
@oznu

### CODE REVIEW

**Comment by oznu** [comment]
Remove dependabot please.

**Comment by donavanbecker** [comment]
> Remove dependabot please.

done.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      ACTIONS_STEP_DEBUG: true
+    steps:
+      - uses: actions/stale@v3.0.17
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+          stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
+          days-before-stale: 10
```
Comment (oznu) [inline .github/workflows/stale.yml:22]: A little low, maybe copy the settings from here:

https://github.com/oznu/homebridge-config-ui-x/blob/master/.github/workflows/stale.yml

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  schedule:
+    - cron: '*/60 * * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_STEP_DEBUG: true
+    steps:
+      - uses: actions/stale@v3.0.17
```
Comment (Supereg) [inline .github/workflows/stale.yml:15]: maybe replace that with just `@v3` so it always uses the latest available?

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: Stale
+
+on:
+  issues:
+    types: [reopened]
+  schedule:
+    - cron: '*/60 * * * *'
```
Comment (Supereg) [inline .github/workflows/stale.yml:7]: this runs every 60 minutes? Is this the default? Not sure if that is maybe a bit overkill?

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - uses: actions/stale@v3.0.17
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+          stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
+          days-before-stale: 10
+          days-before-close: 1
```
Comment (Supereg) [inline .github/workflows/stale.yml:23]: Those limits seem to me a bit short?
@oznu any opinion?

**Review by Supereg — COMMENTED** [review]
Thanks for the PR 👍 
just a few comments

File: .github/workflows/stale.yml (hunk 3 above)
Comment (donavanbecker) [reply .github/workflows/stale.yml]: changed to `'30 1 * * *'`

File: .github/workflows/stale.yml (hunk 2 above)
Comment (donavanbecker) [reply .github/workflows/stale.yml]: done

File: .github/workflows/stale.yml (hunk 4 above)
Comment (donavanbecker) [reply .github/workflows/stale.yml]: just copied ui settings

File: .github/workflows/stale.yml (hunk 1 above)
Comment (donavanbecker) [reply .github/workflows/stale.yml]: copied ui settings

**Review by oznu — APPROVED** [review]
Looks good. Squash and merge to keep the commit history clean.

**Review by Supereg — APPROVED** [review]
Seems good to me 👍

**Pull request merged**